CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Add option to disable color with environment variable "NO_COLOR"
```

Comment:
this line should be under a `### Added` heading while the next should be under `### Changed`

review the [keep a changelog](https://keepachangelog.com/en/1.0.0/) link at the top